### PR TITLE
Add Remote Governance to Crab Parachain

### PR DESCRIPTION
**Pull request by acgxv**

### CODE REVIEW

File: runtime/crab-parachain/src/pallets/remote_governance.rs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+pub struct CrabBestFinalized;
+impl Get<Hash> for CrabBestFinalized {
+	fn get() -> Hash {
+		<pallet_bridge_grandpa::BestFinalized<Runtime, WithCrabGrandpa>>::get()
+	}
+}
+
+frame_support::parameter_types! {
+	pub const CheckInterval: BlockNumber = 2 * HOURS;
```
Comment (hackfisher) [inline runtime/crab-parachain/src/pallets/remote_governance.rs]:
```suggestion
	pub const CheckInterval: BlockNumber = 4 * HOURS;
```

File: runtime/crab-parachain/src/pallets/remote_governance.rs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	for EnsureSpecific
+{
+	type Success = AccountId;
+
+	fn try_origin(o: O) -> Result<Self::Success, O> {
+		o.into().and_then(|o| match o {
+			RawOrigin::Signed(who)
+				if who
+					== array_bytes::hex_into_unchecked(
+						"0x129d025b24257aabdefac93d00419f06a38e3a5e2314dd6866b16e8f205ce074",
```
Comment (acgxv) [inline runtime/crab-parachain/src/pallets/remote_governance.rs]: This is the root account currently in use.

Comment (hackfisher) [inline runtime/crab-parachain/src/pallets/remote_governance.rs]: Whose address?

Comment (acgxv) [reply runtime/crab-parachain/src/pallets/remote_governance.rs]: Or we can register a multisig.

Comment (hackfisher) [reply runtime/crab-parachain/src/pallets/remote_governance.rs]: Let's use the multisig address with same operators

**Review by hackfisher — APPROVED** [review]

File: runtime/crab-parachain/src/pallets/remote_governance.rs (hunk 1 above)
Comment (acgxv) [reply runtime/crab-parachain/src/pallets/remote_governance.rs]: Maybe 1 or 3 days is better?

Comment (hackfisher) [reply runtime/crab-parachain/src/pallets/remote_governance.rs]: I forgot that 1 epoch of crab is 6 hours,

My suggestion is 6 hours or longer, 1 day LGTM.

**Pull request merged**